### DISCUSSION TITLE
Understanding the Types and how they work

### DISCUSSION THREAD

**Discussion by earthshakira** [polypheny/Polypheny-DB#337] [Q&A]
Hi!

While working on [issue 191](https://github.com/polypheny/Polypheny-DB/issues/191). I'm struggling to understand how the types are working in DB. 

What could be a good starting point for understanding how PolyTypes or PolyTypeFamilies work.

Thanks.

**Comment by vogti** [comment]
From the developer perspective, unfortunately, there is no detailed documentation on the type system. For the user perspective we have the following [documentation](ttps://polypheny.org/documentation/Types/).
Where exactly are you struggling and seeking for more information?

**Reply by hennlo** [reply]
Hi @earthshakira Is this still relevant? 
What is the kind of information you are missing?
